packages/react-core/src/components/Label/Label.tsx

```diff
@@ -1,16 +1,73 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Label/label';
+import { Button } from '../Button';
 import { css } from '@patternfly/react-styles';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
 
 export interface LabelProps extends React.HTMLProps<HTMLSpanElement> {
   /** Content rendered inside the label. */
   children: React.ReactNode;
   /** Additional classes added to the label. */
   className?: string;
+  /** Color of the label. */
+  color?: 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey';
+  /** Variant of the label. */
+  variant?: 'outline' | 'filled';
+  /** Icon added to the left of the label text. */
+  icon?: React.ReactNode;
+  /** Close click callback for removable labels. If present, label will have a close button. */
+  onClose?: (event: React.MouseEvent) => void;
+  /** Node for custom close button. */
+  closeBtn?: React.ReactNode;
+  /** Additional properties for the default close button. */
+  closeBtnProps?: any;
+  /** Href for a label that is a link. If present, the label will change to an anchor element. */
+  href?: string;
 }
 
-export const Label: React.FunctionComponent<LabelProps> = ({ children, className = '', ...props }: LabelProps) => (
-  <span {...props} className={css(styles.label, className)}>
-    {children}
-  </span>
-);
+const colorStyles = {
+  blue: styles.modifiers.blue,
+  cyan: styles.modifiers.cyan,
+  green: styles.modifiers.green,
+  orange: styles.modifiers.orange,
+  purple: styles.modifiers.purple,
+  red: styles.modifiers.red,
+  grey: ''
+};
+
+export const Label: React.FunctionComponent<LabelProps> = ({
+  children,
+  className = '',
+  color = 'grey',
+  variant = 'filled',
+  icon,
+  onClose,
+  closeBtn,
+  closeBtnProps = {
+    'aria-label': 'label-close-button'
```

Comment:
Try:
```
closeBtnProps = {
    'aria-label': 'label-close-button',
    ...closeBtnProps
}
```
or else when a consumer specifies just one property other than `aria-label` the `aria-label` won't be set. If that syntax doesn't work, on line 56 `{...{'aria-label': 'label-close-button', ...closeBtnProps}}` should.